applications/targets/tests/test_target_delete.py

```diff
@@ -0,0 +1,66 @@
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+from rest_framework.views import status
+
+from .factories import TargetFactory
+from applications.users.tests.factories import UserFactory
+from applications.targets.models import Target
+
+
+class TargetDeleteTest(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.url = reverse('target-list')
+
+    def call_targets_delete(self):
+        return self.client.delete(f'{self.url}{self.pk}/')
+
+    def setUp(self):
+        self.user = UserFactory(confirmed=True)
+
+    def test_no_created_targets_responds_not_found(self):
+        self.client.force_authenticate(user=self.user)
+        self.pk = 1
```

Comment:
I would remove `pk` attribute from this class, and would pass the pk to `call_targets_delete`.
It's only a suggestion. It's more clear and understandable.